no "we" (#415)
TYPO3-Documentation/TYPO3CMS-Book-ExtbaseFluid: Documentation/4-FirstExtension/1-the-example-extension.rst

The example extension
=====================

The first extension will show an inventory list of products,

> which can be created in the backend list module.

Reviewer: ```suggestion
which is maintained in the backend *List* module.
```

This hopefully makes it clear that not the extension is created in the backend List module.

   * Define the database tables
   * Configure the display of the backend forms

>    * Create the repository for the product object

Reviewer: Actually `objects` is correct here since this is barely about a single instance.

Alternatively use something like _`Product` type_.
